### DISCUSSION TITLE
Default voices.

### DISCUSSION THREAD

**Discussion by anycolorsmith** [probonopd/MiniDexed#193] [Q&A] [answered]
Hi, DX guys!
Good. The issue of importing banks was clarified. But. What voices are played when we don't have banks on the memory card?

**Comment by craigyjp** [comment, marked as answer]
The standard rom 1A regardless of banks.

**Reply by anycolorsmith** [reply]
> The standard rom 1A regardless of banks.

Can i change this?

**Comment by craigyjp** [comment]
Pretty sure once you put a new bank at sysex/voice/000000_filename.syx then
these becomes the default sounds.

**Reply by probonopd** [reply]
Yes.
